_config.yml.example

```diff
@@ -73,6 +73,10 @@ comment:
 # Share
 share: default # options: jiathis, bdshare, addtoany, default
 
+#Count
```

Comment:
thanks for your PR, but in my opinion, `busuanzi` is a site pv counter which is similar to `google analytics`, 

so, maybe it should be contained in the tag `plugins`, rather than `Count` in root.

like this:
```markdown
# Plugins
plugins:
    ...
    busuanzi_count: true # [true, false] | is enable busuanzi site pv statistics
```

and use it in ejs with `theme.plugins.busuanzi_count`.

what do you think?

thanks a lot again.

Reply:
I agree with you, it should be contained in the tag `plugins`.